backend/src/routes.js

```diff
@@ -1,32 +1,52 @@
 const express = require('express');
-const {User} = require("./Schema");
+const { User, NGO } = require("./Schema");
 const router = express.Router();
 
+//Route to get all users
+router.get('/users', async (req, res) => {
+    const users = await User.find({});
 
-router.get('/users', (req, res) => {
-    res.send(`<h1>No users yet :(</h1>`)
+    try {
+        res.status(200).json(users)
```

Comment:
Please wrap this response and the error response in an object. Check the endpoint below ;)